### ISSUE TITLE
Use of undeclared Var zprint.config/java

### ISSUE THREAD

**Issue by daaku** [kkinnear/zprint#92]
I'm getting this warning in a project being compiled using `shadow-cljs` -- looks like a case of JVM specific code without a conditional in a cljc file.

```
------ WARNING #1 --------------------------------------------------------------
 File: zprint/config.cljc:1166:13
--------------------------------------------------------------------------------
1163 |   (let [; Fix file-sep for Windows file separator regex-file-sep
1164 |         regex-file-sep (if (= file-sep "\\") "\\\\" file-sep)
1165 |         file-sep-pattern (re-pattern regex-file-sep)
1166 |         cwd (java.io.File. ".")
-------------------^------------------------------------------------------------
 Use of undeclared Var zprint.config/java
--------------------------------------------------------------------------------
1167 |         path-to-root (.getCanonicalPath cwd)
1168 |         dirs-to-root (clojure.string/split path-to-root file-sep-pattern)]
1169 |     (reduce (partial get-config-from-dirs filename-vec file-sep)
1170 |       ["."]
--------------------------------------------------------------------------------
```

This is in version `0.4.14`.

**Comment by kkinnear** [comment]
Tested things in the wrong order.  Sigh.  This is fixed now in `0.4.15`.  Thanks for noticing so quickly and letting me know.

**Comment by daaku** [comment]
Thanks for the quick fix -- no more warnings. 👍
